apps/web/src/views/FixedStaking/components/AprCell.tsx

```diff
@@ -17,6 +32,7 @@ export default function AprCell({
 }) {
   return selectedPeriodIndex === null || !selectedPool ? (
     <Flex alignItems="center">
+      {pool.token.chainId === bsc.id && pool.token.symbol === 'WBNB' ? <WBNBGiftLink /> : null}
```

Comment:
how about use the token equal method here?

Reply:
pool.token.equals(bscTokens.bnb)
something like this

Reply:
you mean import bscTokens?

Reply:
yes, do you think it's a good idea? 
if it's not, it's ok, just found the method usage from the Chef Jackson's previous PR